CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## [1.6.2] - 2022-04-26
```

Comment:
(amend this to the commit instead of accepting this suggestion)

```suggestion
## [1.6.2] - 2022-04-27
```

Reply:
Done